app/views/users/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Create an Account </h1>
+<div id = "new-user-form">
+  <%= form_for :new_user, url: '/users' do |t| %>
```

Comment:
Your controller sets `@user = User.new` so here you could do just `form_for @user` and let rails infer the url etc.